Dockerfile.runtime

```diff
@@ -23,6 +23,7 @@ RUN apt clean && rm -rf /var/lib/app/lists/*
 
 # Install libraries
 COPY --from=build /opt/overte/libraries/*/*.so /lib/
+COPY --from=build /root/overte-files/vcpkg/77b4e5f4/installed/x64-linux/lib/libnode.so.108 /lib/
```

Comment:
This looks troublesome.

The absolute /root path seems like a bad idea, since with podman there's no reason why anything would need to run as root.

Also, 77b4e5f4 is not going to be constant. This will be out of date as soon as we change anything in vcpkg.

Reply:
I have fixed this in the pr's below